man/ff-deps.1.md

```diff
@@ -0,0 +1,125 @@
+% FF-DEPS(1) % Daniel de Kok, Sebastian Pütz % Apr 6, 2019
+
+NAME ====
```

Comment:
Didn't notice this earlier: the equals should be below the section names.

Reply:
That's because I (or vim?) messed up auto-wrapping lines. 

I set `textwidth = 80` and used `gq` to wrap lines. That put all equals-signs below their headings.

Reply:
Must be vim settings, but I use gqip to rewrap lines and it only rewraps lines from the paragraph the cursor is in. My vim configuration is more than a decade old, so I don't know what is responsible for proper wrapping ;):

https://github.com/danieldk/dotfiles/blob/master/.vimrc

Reply:
(It might be just `gqip`, which wraps inner paragraphs.)